migrations/808-waffle-automatic-validation.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO waffle_flag (name, everyone, percent, testing, superusers, staff, authenticated, languages, rollout, note, created, modified)
+       VALUES ('automatic-validation', 0, NULL, 0, 0, 0, 0, '', 0, "Guards the automatic validation/review/signing or unlisted add-ons when they're submitted.", NOW(), NOW());
```

Comment:
I wanted to use the switch at first, and then thought about the flag's possibility to add some users (or maybe just set it for staff, or superusers...) to test on production. I think this would be valuable (but now, I don't think we'll use the "roll out" feature).